client/extensions/woocommerce/index.js

```diff
@@ -178,10 +170,7 @@ function redirectIfWooCommerceNotInstalled( context, next ) {
 	next();
 }
 
-export default async function ( _, addReducer ) {
-	await addReducer( [ 'extensions', 'woocommerce' ], reducer );
-	installActionHandlers();
-
```

Comment:
Has this been obsoleted by the state modularization that adds the reducer automatically when a related action/selector is imported?

Reply:
Oh, you're removing the _entire_ Woo state, so the reducer and data layer handlers no longer exist 😨 I didn't notice that at first.

Reply:
Yes 😅  Maybe the old title of the PR caused the confusion, I updated it.